recipe/meta.yaml

```diff
@@ -130,3 +131,4 @@ extra:
     - dbast
     - jezdez
     - kenodegard
+    - beeankha
```

Comment:
@beeankha guessing you are happy to be a maintainer here 🙂

Reply:
Affirmative! 😄